src/BLSSignatureChecker.sol

```diff
@@ -15,13 +17,13 @@ import {BN254} from "./libraries/BN254.sol";
  * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
  * @notice This is the contract for checking the validity of aggregate operator signatures.
  */
-contract BLSSignatureChecker is IBLSSignatureChecker {
+contract BLSSignatureChecker is Initializable, IBLSSignatureChecker {
```

Comment:
I am somewhat worried about the storage layout change here.
Can we do without this import? I think the only way this storage layout is backwards-compatible is if you're upgrading a contract that inherits from Initializable directly before inheriting from this contract? Seems like a large potential footgun to get an initializer for a single boolean

Reply:
makes sense, anything that is also initializable shouldn't have shift, but i understand the concern

Reply:
the linearization of the inheritance is ultimately what matters, so just inheriting Initializable from elsewhere isn't a guarantee either 😞

Reply:
yeah that's true, seems like it could be a foot gun